pillow_heif/_pillow_heif.c

```diff
@@ -704,7 +704,6 @@ static PyObject* _CtxWriteImage_create(CtxWriteObject* self, PyObject* args) {
     CtxWriteImageObject* ctx_write_image = PyObject_New(CtxWriteImageObject, &CtxWriteImage_Type);
     if (!ctx_write_image) {
         heif_image_release(image);
-        PyErr_SetString(PyExc_RuntimeError, "could not create CtxWriteImage object");
         return NULL;
```

Comment:
I suggest to use `return PyErr_NoMemory();` everywhere where these is no memory, what do you think?

ref: https://docs.python.org/3.8/c-api/exceptions.html#c.PyErr_NoMemory

Reply:
never mind, looking at Python sources it is already done: https://github.com/python/cpython/blob/4197a796ecf3a751ad7245b8d4f980d6d444b614/Objects/listobject.c#L210

Reply:
Yes, I have linked to Python sources in the corresponding commit messages that sets these exceptions.